src/ui/components/atoms/ol-item/OLItem.stories.tsx

```diff
@@ -1,9 +1,20 @@
 import type { Meta, StoryObj } from '@storybook/react';
+import { ThemeProvider } from '@mui/material/styles';
+import { lightTheme } from '../../../../style-library/themes/default';
 import OLItem from './OLItem';
 
 const meta = {
   title: 'ui/components/atoms/ol-item',
   component: OLItem,
+  decorators: [
+    (Story) => (
+      <ThemeProvider theme={lightTheme}>
```

Comment:
It is not necessary to call ThemeProvider at this level. The theme is added globally for all stories